app/tests/test_reporter.py

```diff
@@ -0,0 +1,93 @@
+import os
+from unittest.mock import MagicMock
```

Comment:
Why use unittest?

Reply:
this is what pytest MockerFixture returns